src/main/java/uk/gov/cslearning/record/service/UserRecordService.java

```diff
@@ -29,77 +25,37 @@
+                             LearningCatalogueService learningCatalogueService) {
         checkArgument(courseRecordRepository != null);
-        checkArgument(xApiService != null);
         checkArgument(learningCatalogueService != null);
         this.courseRecordRepository = courseRecordRepository;
-        this.xApiService = xApiService;
-        this.learningCatalogueService = learningCatalogueService;
-        this.collectionsService = collectionsService;
-        this.registryService = registryService;
     }
 
     public Collection<CourseRecord> getUserRecord(String userId, List<String> activityIds) {
         LOGGER.debug("Retrieving user record for user {}, activities {}", userId, activityIds);
 
         Collection<CourseRecord> courseRecords = courseRecordRepository.findByUserId(userId);
 
-        LocalDateTime since = courseRecords.stream()
-                .map(CourseRecord::getLastUpdated)
-                .filter(Objects::nonNull)
-                .reduce((a, b) -> a.isAfter(b) ? a : b)
-                .orElse(null);
-
-        try {
-            Collection<Statement> statements = xApiService.getStatements(userId, null, since);
-
-            StatementStream stream = new StatementStream(learningCatalogueService, registryService);
-
-            Collection<CourseRecord> updatedCourseRecords = stream.replay(statements,
-                    statement -> ((Activity) statement.getObject()).getId(),
-                    courseRecords);
-
-            courseRecordRepository.saveAll(updatedCourseRecords);
-
-            for (CourseRecord courseRecord : updatedCourseRecords) {
-                if (!courseRecords.contains(courseRecord)) {
-                    courseRecords.add(courseRecord);
-                }
-            }
-
-            if (activityIds != null && !activityIds.isEmpty()) {
-                return courseRecords.stream()
-                        .filter(courseRecord -> activityIds.stream().anyMatch(courseRecord::matchesActivityId))
-                        .collect(Collectors.toSet());
-            }
-
-            return courseRecords;
-        } catch (IOException e) {
-            throw new RuntimeException("Exception retrieving xAPI statements.", e);
+        if (activityIds != null && !activityIds.isEmpty()) {
+            return courseRecords.stream()
+                    .filter(courseRecord -> activityIds.stream().anyMatch(courseRecord::matchesActivityId))
+                    .collect(Collectors.toSet());
         }
+
+        return courseRecords;
     }
 
     @Transactional
     public void deleteUserRecords(String uid) {
-        collectionsService.deleteAllByLearnerUid(uid);
         courseRecordRepository.deleteAllByUid(uid);
     }
 
     @Transactional
-    public void deleteOldRecords(DateTime dateTime, LocalDateTime localDateTime) {
-        collectionsService.deleteAllByAge(dateTime);
+    public void deleteOldRecords(LocalDateTime localDateTime) {
```

Comment:
Naming might cause confusion when calling the method. Maybe something like `deleteRecordsLastUpdatedBefore(localDateTime)` would be more suitable?